airbyte-webapp/src/components/ui/Flex/FlexContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from "classnames";
+import React, { HTMLAttributes } from "react";
+
+import styles from "./FlexContainer.module.scss";
+
+interface FlexContainerProps {
+  className?: string;
+  /**
+   * The flex-direction css property
+   */
+  direction?: "row" | "column" | "row-reverse" | "column-reverse";
```

Comment:
That would allow the prop to be set to ` "-moz-initial" | "inherit" | "initial" | "revert" | "unset"` as well, but these are not actually applied by the current logic. Besides that I liked the explicit nature of the interface. Let me know what you think.

Reply:
Sounds reasonable enough to me!
Doing a `Pick` on it to just get those props would be pointless I think :)